# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: startupcache/StartupCache.cpp
# Commit: 4208ed5fc3b0
# Full Hash: 4208ed5fc3b02fddc3624805ff3937a68a4aa4f6
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2020-07-08 09:42:17
# Regressor Bug: 1627075
# File Overlap Count: 1
# Description:
#   Bug 1627075 - Build Omnijar file list from startup cache r=froydnj
#   
#   We would like to be able to defer opening the omnijar files until after startup
#   if the StartupCache has already been populated. Opening the omnijar files takes
#   a nontrivial time, at least on Windows, and almost everything in the omnijar
# ==============================================================================

diff -r 14420a624d0a -r 4208ed5fc3b0 startupcache/StartupCache.cpp
--- a/startupcache/StartupCache.cpp	Tue Jul 07 15:19:09 2020 +0000
+++ b/startupcache/StartupCache.cpp	Wed Jul 08 02:43:02 2020 +0000
@@ -439,7 +439,7 @@
   return NS_OK;
 }
 
-// Makes a copy of the buffer, client retains ownership of inbuf.
+// Takes ownership of the input buffer
 nsresult StartupCache::PutBuffer(const char* id, UniquePtr<char[]>&& inbuf,
                                  uint32_t len) {
   NS_ASSERTION(NS_IsMainThread(),